frontend/src/components/navbar/Navbar.js

```diff
@@ -49,11 +49,29 @@ const useStyles = makeStyles(theme => ({
 	},
 	toolbar: {
 		height: 300,
-		marginRight: "20rem",
-		marginLeft: "2rem"
+		marginRight: "2rem",
+		marginLeft: "-8rem",
```

Comment:
This causes the horizontal margins on the navbar to be asymmetrical on large screens.

Reply:
![image](https://user-images.githubusercontent.com/80652363/152048787-36081b66-d511-44ad-9175-3c93dcf3ba26.png)
Com simetria:
![image](https://user-images.githubusercontent.com/80652363/152049002-3cc17be7-fecc-4f58-859f-4d7ac4748ef2.png)
Sem simetria:
![image](https://user-images.githubusercontent.com/80652363/152049083-1d434185-c6c2-41ee-9f30-ee0d95144b7f.png)

Honestamente prefiro a assimetria, dá mais espaço para os lados

Reply:
Mantém em large screen mas pode ficar como entenderes para medium e small